scripts/generate-bosh-lite-manifest

```diff
@@ -83,7 +83,12 @@ if [[ "$(echo "$BOSH_STATUS" | grep Name)" != *"$EXPECTED_DIRECTOR_NAME"* ]]; th
   exit 1
 fi
 
-tmpdir=$(mktemp -d /tmp/autoscaler-bosh-lite-manifest.XXXXX)
+deployments_dir=$(cd $(dirname $(dirname $0)) && pwd)/deployments
```

Comment:
diego-release and cf-release use the path 'bosh-lite/deployments'. Should we follow the same way?